lib/DXIL/DxilModule.cpp

```diff
@@ -96,29 +96,13 @@ void ClearDxilHook(Module &M);
 //  DxilModule methods.
 //
 DxilModule::DxilModule(Module *pModule)
-: m_StreamPrimitiveTopology(DXIL::PrimitiveTopology::Undefined)
-, m_ActiveStreamMask(0)
-, m_Ctx(pModule->getContext())
+: m_Ctx(pModule->getContext())
 , m_pModule(pModule)
-, m_pEntryFunc(nullptr)
-, m_EntryName("")
 , m_pMDHelper(make_unique<DxilMDHelper>(pModule, make_unique<DxilExtraPropertyHelper>(pModule)))
 , m_pDebugInfoFinder(nullptr)
-, m_pSM(nullptr)
-, m_DxilMajor(DXIL::kDxilMajor)
-, m_DxilMinor(DXIL::kDxilMinor)
-, m_ValMajor(1)
-, m_ValMinor(0)
-, m_ForceZeroStoreLifetimes(false)
 , m_pOP(make_unique<OP>(pModule->getContext(), pModule))
 , m_pTypeSystem(make_unique<DxilTypeSystem>(pModule))
-, m_bDisableOptimizations(false)
-, m_bUseMinPrecision(true) // use min precision by default
-, m_bAllResourcesBound(false)
-, m_IntermediateFlags(0)
-, m_AutoBindingSpace(UINT_MAX)
 , m_pSubobjects(nullptr)
```

Comment:
For this and `m_pDebugInfoFinder`, you can remove the explicit `nullptr` initialization - since they're `unique_ptr`s, initialization with `nullptr` is the default.